src/stories/Library/material-description/MaterialDescription.tsx

```diff
@@ -66,8 +66,12 @@ const MaterialDescription: React.FC<MaterialDescriptionProps> = ({
   return (
     <section className="material-description">
       <h2 className="text-header-h4 pb-24">Beskrivelse</h2>
-      <p className="text-body-large ">{description}</p>
-      <div className="material-description__links mt-32">{seriesLines}</div>
+      <p className="text-body-large material-description__content">
+        {description}
+      </p>
+      <div className="material-description__links mt-32">
+        {HorizontalTermLines}
```

Comment:
It's a matter of taste but I think it would be more clear to have the map loop here.
I had to investigate how the variable was constructed before I understood what was happening.
And it is the approach that we most commonly use.
So something like:
```tsx
{horizontalTermLines.map((item) => <HorizontalTermLine {...item} />)}